item-api/src/main/java/com/example/itemapi/global/config/CacheConfig.java

```diff
@@ -0,0 +1,75 @@
+package com.example.itemapi.global.config;
+
+import com.example.itemapi.global.utils.ObjectMapperUtils;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.databind.jsontype.BasicPolymorphicTypeValidator;
+import com.fasterxml.jackson.databind.jsontype.PolymorphicTypeValidator;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.github.benmanes.caffeine.cache.Caffeine;
+import lombok.RequiredArgsConstructor;
+import org.springframework.cache.CacheManager;
+import org.springframework.cache.annotation.EnableCaching;
+import org.springframework.cache.caffeine.CaffeineCache;
+import org.springframework.cache.support.SimpleCacheManager;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import org.springframework.data.redis.cache.RedisCacheConfiguration;
+import org.springframework.data.redis.cache.RedisCacheManager;
+import org.springframework.data.redis.connection.RedisConnectionFactory;
+import org.springframework.data.redis.serializer.GenericJackson2JsonRedisSerializer;
+import org.springframework.data.redis.serializer.RedisSerializationContext;
+import org.springframework.data.redis.serializer.StringRedisSerializer;
+
+import java.util.Arrays;
+import java.util.List;
+
+@RequiredArgsConstructor
+@Configuration
+@EnableCaching
+public class CacheConfig {
+
+    private final RedisConnectionFactory redisConnectionFactory;
+
+    @Primary
```

Comment:
```suggestion
    @Primary
```

`localCacheManager` 가  primary 로 우선순위를 가져가게 될텐데요

`CachePut, CacheEvict` 과 같이 local, redis 캐시를 갱신해야하는 상황일때

둘 다 적용되는지 확인 부탁드려요~!